release_creation/bundle/requirements/v1.8.pre.requirements.txt

```
@@ -12,6 +12,7 @@ dbt-trino~=1.8.0b1
 # `dbt-fabric==1.8.3` breaks `dbt-synapse`
 dbt-fabric>=1.8.0rc3,<1.8.3
 dbt-synapse~=1.8.0rc1
+dbt-athena-community~=1.8.0rc1
```

as of this morning,  `dbt-athena-community 1.8.1` is out (PyPI GitHub Community Slack announcement)
```suggestion
dbt-athena-community~=1.8.1
```